test/e2e/mcd_test.go

```diff
@@ -425,6 +425,39 @@ func TestReconcileAfterBadMC(t *testing.T) {
 	}
 }
 
+func TestFullyDeleteMCShippedFiles(t *testing.T) {
+	cs := framework.NewClientSet("")
```

Comment:
A good sanity check to do upfront here is that `/etc/something` *doesn't* exist (I mean, it shouldn't, but it helps make it clear how this test works :) )

Reply:
I'd add that thanks @jlebon !

Reply:
added